chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -19,6 +19,12 @@ extension NetworkAPI {
 	func delete(client: Client, completion: @escaping NetworkCompletion) {
 		performDeleteRequest(to: "clients/\(client.id)", completion: completion)
 	}
+
+	func addClient(name: String, completion: @escaping NetworkCompletion) {
+		let header = ["Content-Type" : "application/json; charset=utf-8"]
+		let body = ["name" : name]
```

Comment:
JSON (or rather data structure creation) should be done in the Client class, and passed on.
You can also see what JSON type data should look like (in terms of Swift type).

a.k.a.: pass in a Client object, and the Client class should have a `toJSON` method